.golangci.yaml

```diff
@@ -0,0 +1,146 @@
+run:
+  timeout: 5m
+  go: "1.19"
+  skip-files:
+    - "zz_generated.*\\.go$"
+  allow-parallel-runners: true
+
+linters:
+  disable-all: true
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default-linters
+  enable:
```

Comment:
I saw that there's a [similar .golangci.yml](https://github.com/kubernetes-sigs/cluster-api/blob/main/.golangci.yml) in Cluster API. Do you think we can try to follow their format so the code the linter accepts is consistent?

Reply:
I added missing linters from their configuration and fixed new issues. For now I decided to comment out `revive` linter because it will require a lot of changes (I'll add them in a separate PR).